### PR TITLE
Imp wave2 845 grad details outcomes next steps

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 1]
```diff
@@ -64,6 +64,7 @@
 <div class="outcomes">
 <?php print render($content['field_fgs_grad_degree_outcomes']);?>
 </div>
+  <?php if (isset($content['field_fgs_grad_degree_acdmc_path'])) { ?>
 <div class="ac_path">
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Appears (on github anyway) that there are some indenting issues here. Can you correct those?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module [hunk 2]
```diff
@@ -43,5 +43,13 @@ function ucgy_fgs_form_alter(&$form, &$form_state, $form_id) {
       $form['field_fgs_specializations']['und']['add_more']['#prefix'] = '<div style="display:none;">';
       $form['field_fgs_specializations']['und']['add_more']['#suffix'] = '</div>';
     }
+
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: Can you add what these variables are in comments?

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module [hunk 3]
```diff
@@ -43,5 +43,13 @@ function ucgy_fgs_form_alter(&$form, &$form_state, $form_id) {
...
+    $ac_path_title = $form['field_fgs_grad_degree_acdmc_path']['und']['#title'];
+    $ac_path_desc = $form['field_fgs_grad_degree_acdmc_path']['und']['#description'];
+
+   $form['field_fgs_grad_degree_acdmc_path']['und']['#prefix'] = '<p><strong>' . $ac_path_title . '</strong><br/>' . $ac_path_desc . '</p>' ;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: Wrap these text in t()

**Comment by dpfitzsi** [comment]
FYI Mike you will need to resolve conflicts with the ucgy_fgs module before this can be merged.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_supervisor/templates/node--fgs-supervisor.tpl.php [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+<!-- If the image is set it will show up here-->
+<?php if (isset($content['field_fgs_supervisor_image'])):?>
+<div><?php print render($content['field_fgs_supervisor_image']); ?></div>
+<?php endif; ?>
+
+<!-- first name, last name, and degrees supervised are required so no if statment is needed -->
+<div>
+    <?php
+
+        // If the a website url was entered
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs_supervisor/templates/node--fgs-supervisor.tpl.php]: This was fixed by Robb in Jira 826

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module (hunk 3 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: fixed. moved to PR https://github.com/ucalgary/www.ucalgary/pull/464

**Comment by mhughes2012** [comment]
closing this one to avoid merging conflicts. Moved to https://github.com/ucalgary/www.ucalgary/pull/464

**Pull request closed without merging**